app/controllers/agent/ManageAgentController.scala

```diff
@@ -84,6 +93,7 @@ class ManageAgentController @Inject()(
                             .getMyAgentPropertyLinks(agentCode, params, PaginationParams(1, 100, true))
         agentDetails <- agentRelationshipService.getAgentNameAndAddress(agentCode)
         myAgents     <- agentRelationshipService.getMyOrganisationAgents()
+        listYears = myAgents.agents.flatMap(_.listYears).headOption
```

Comment:
This will return a sequence of list years that a user's agent can act upon, but only for the first agent in their list? Is that correct?

Looking at how you've used this value in the view:
```
    listYears.fold("Rating lists for this property are not available yet") { years =>
        years match {
            case Seq(listYear) => messages("propertyRepresentation.ratingList.p", listYear)
            case _ => messages("propertyRepresentation.ratingLists.p")
        }
    }
 ```
It will display "Rating lists for this property are not available yet" if your agents cannot act on any of your list years. I might be misunderstanding the requirement, but is this correct?